dandiapi/api/templates/api/account/base.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+
+  <title>DANDI Archive</title>
+
+  <style>
+    .container {
+      position: absolute;
+      left: 50%;
+      top: 50%;
+      -webkit-transform: translate(-50%, -50%);
+      transform: translate(-50%, -50%);
+    }
+    .input-field {
+      margin-top: 10%;
+      margin-bottom: 10%;
+    }
+    .logo-container {
+      padding-left: 2%;
+      margin: auto;
+    }
+    .logo {
+      height: 40px;
+      margin-top: 10px;
+    }
+  </style>
+</head>
+
+<body>
+  <nav style="background-color: #f5f5f5;">
+    <div class="nav-wrapper logo-container">
+      <a href="https://dandiarchive.org" class="brand-logo">
+        {% load static %}
+        <img src="{% static 'api/dandi_logo.svg' %}" class="logo">
```

Comment:
Could we not link in the logo from https://github.com/dandi/dandi.github.io/blob/master/assets/dandi_logo.svg instead?